app/controllers/api/v1/knowledge_base_status_controller.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+class Api::V1::KnowledgeBaseStatusController < ApplicationController
+  include JWTHelper
+
+  skip_before_action :verify_authenticity_token
+  before_action :authenticate_jwt
+
+  def build_knowledge_base
+    conn = KbConnection.kb_rmq_connection
+    exchange = KbConnection.get_exchange(conn)
+    kb = KnowledgeBaseStatus.where(project_id: params[:project_id]).first_or_initialize
+    puts "KBupdater Cron Job: converting and sending #{knowledge_base_status.project_id}"
+    exchange.publish(kb.json_message, key: ENV['KB_EXCHANGE_KEY'])
+    puts "Message displayed successfully"
+    conn.close
+  rescue StandardError => e
+    Rails.logger.info(e.message)
+    Airbrake.notify(e)
+    conn.close
+    raise StandardError, e.message
```

Comment:
No need to raise `StandardError` here. 
Since this is an api call, this function should respond with a json message and appropriate status code.